src/Windows/Markdown/Editor/Impl/Preview/Code/RCodeEvaluator.cs

```diff
@@ -82,10 +82,11 @@ private string FormatError(string error)
             _errors = new StringBuilder();
 
             try {
-                using (var inter = await session.BeginInteractionAsync(isVisible: false, cancellationToken: cancellationToken)) {
+                //await session.ExecuteAsync(expression, REvaluationKind.Mutating, cancellationToken: cancellationToken);
```

Comment:
Undoing previous change which was incorrect. ExecuteAsync does not work here. Removing the comment.